modules/single-port-sg/main.tf

```diff
@@ -46,24 +46,24 @@ locals {
 
 # ingress rule for tcp, if enabled
 resource "aws_security_group_rule" "tcp-ingress" {
-  count             = "${local.tcp}"
+  count             = local.tcp
   type              = "ingress"
   description       = "${var.description} (tcp)"
-  from_port         = "${var.port}"
-  to_port           = "${var.port}"
+  from_port         = var.port
+  to_port           = var.port
   protocol          = "tcp"
   cidr_blocks       = ["${var.cidr_blocks}"]
```

Comment:
this one should change as well

Reply:
Yes, definitely. Good catch, I have updated that.